cassandra/src/main/scala/akka/stream/alpakka/cassandra/javadsl/CassandraSink.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.cassandra.javadsl
+
+import java.util.concurrent.CompletionStage
+import java.util.function.BiFunction
+
+import akka.Done
+import akka.stream.javadsl.Sink
+import com.datastax.driver.core.{ BoundStatement, PreparedStatement, Session }
+import akka.stream.alpakka.cassandra.scaladsl.{ CassandraSink => ScalaCSink }
+
+import scala.compat.java8.FutureConverters._
+
+object CassandraSink {
+
+  def create[T](parallelism: Int,
+                statement: PreparedStatement,
+                statementBinder: BiFunction[T, PreparedStatement, BoundStatement],
+                session: Session): Sink[T, CompletionStage[Done]] = {
+    val sink = ScalaCSink.apply[T](parallelism, statement, (t, p) => statementBinder.apply(t, p))(
+      session,
+      scala.concurrent.ExecutionContext.global)
```

Comment:
Take the EC as a parameter to `create` instead, just like the Scala api does (if done like this the user has no control).